.github/workflows/latest-release.yml

```diff
@@ -35,11 +44,32 @@ jobs:
         run: |
           ./tests/test-scenarios-github.sh
 
+      - name: Check Tag
+        id: check-tag
+        run: |
+          if [[ ${{ github.event.ref }} =~ ^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+              echo ::set-output name=match::true
+          fi
+
+      - name: Create Release
+        if: steps.check-tag.outputs.match == 'true'
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
```

Comment:
Where is this GITHUB_TOKEN used after passing it as env var?

Reply:
Currently we are not using this token. It is automatically generated by Github and can be used for authenticating the workflow run. I think it's currently not needed, will remove this. Thanks for pointing.